specified units on response values in bones or DC

samgutentag opened the pull request:
fix for helium/blockchain-http#434

samgutentag committed:
specified units on response values in bones or DC

cokes518 commented on docs/api/blockchain/introduction.mdx:
```diff
@@ -53,19 +52,26 @@ next page:
+| `balance`    | bones        | `100,000,000 bone` to `1 HNT` |
+| `stake`      | bones        | `100,000,000 bone` to `1 HNT` |
+| `sum`        | bones        | `100,000,000 bone` to `1 HNT` |
+| `fee`        | Data Credits | `100,000 DC` to `1 HNT`       |
```
this doesn't seem right. The price of Data Credits is fixed in USD (1 Data Credit = $0.00001). so 100,000 DC = $1 USD

cokes518 commented on docs/api/blockchain/introduction.mdx:
```diff
@@ -53,19 +52,26 @@ next page:
+| `stake`      | bones        | `100,000,000 bone` to `1 HNT` |
+| `sum`        | bones        | `100,000,000 bone` to `1 HNT` |
+| `fee`        | Data Credits | `100,000 DC` to `1 HNT`       |
+| `dc_balance` | Data Credits | `100,000 DC` to `1 HNT`       |
```
same thing here

cokes518 commented on docs/api/blockchain/validators.mdx:
````diff
@@ -6594,7 +6594,7 @@ GET https://api.helium.io/v1/validators/stats
 ```
 
-Returns stats for validators
+Returns stats for validators.
````
```suggestion
Returns stats for Validators.
```

cokes518 commented on docs/api/blockchain/validators.mdx:
```diff
@@ -7584,6 +7590,12 @@ _Query Parameters_
 
 Rewards for a validator in the last 24 hours: `min_time=-24%20hour`
```
```suggestion
Rewards for a Validator in the last 24 hours: `min_time=-24%20hour`
```

samgutentag committed:
Apply suggestions from code review

Co-authored-by: Coco Tang <coco@helium.com>

samgutentag committed:
DC o USD typo corrected.

The price of Data Credits is fixed in USD (1 Data Credit = $0.00001). so 100,000 DC = $1 USD

samgutentag committed:
Merge branch 'gutentag/transaction-fee-units' of https://github.com/helium/docs into gutentag/transaction-fee-units

samgutentag commented:
DC to USD conversion corrections applied

samgutentag commented:
closing PR and incorporating changes in broader docs cleanup efforts
